android/quest/src/main/java/org/smartregister/fhircore/quest/data/patient/PatientRepository.kt

```diff
@@ -149,7 +149,8 @@ constructor(
 
       result.map {
         QuestionnaireConfig(
-          appId = configurationRegistry.appId,
+          appId =
+            if (configurationRegistry.isAppIdInitialized() || configurationRegistry.appId != null) configurationRegistry.appId else "",
```

Comment:
I don't think we should pass an empty string, can we change QuestionnaireConfig so that it can handle a null being passed here?

That way users of QuestionnaireConfig don't need to remember this check